packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -862,7 +862,7 @@ class DatePicker extends React.Component {
               : null
           }
           {
-            this.state.open || this.props.inline
+            (this.state.open && !this.props.disabled) || this.props.inline
```

Comment:
Can we add a test to test this when it is open and then disable it?

Reply:
Added